sdks/python/apache_beam/utils/annotations_test.py

```diff
@@ -32,9 +32,10 @@ def test_deprecated_with_since_current_message(self):
       @deprecated(since='v.1', current='multiply', extra_message='Do this')
       def fnc_test_deprecated_with_since_current_message():
         return 'lol'
+      warnings.simplefilter("once")
```

Comment:
Do we actually need this line? If the test does not pass with out it, most likely we need to fix the test.